packages/client/src/parse-callback.test.ts

```diff
@@ -11,6 +11,6 @@ describe('parseRedirectCallback', () => {
   });
 
   test('no query params should fail', () => {
-    expect(() => parseRedirectCallback('http://localhost:3000')).toThrow();
+    expect(parseRedirectCallback('http://localhost:3000')).toHaveProperty('error');
```

Comment:
this test says 'no query params should fail' and we expect the  `parseRedirectCallback('http://localhost:3000')` return an object with property `error`?

Reply:
See [the discussions on Slack](https://silverhand-io.slack.com/archives/C02A8G4HVAM/p1636700239005200)